plugins/huaweicloud/src/main/java/org/eclipse/xpanse/plugins/huaweicloud/resources/SubnetInfo.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.plugins.huaweicloud.resources;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.datatype.jsr310.ser.OffsetDateTimeSerializer;
+import com.huaweicloud.sdk.vpc.v2.model.ExtraDhcpOption;
+import com.huaweicloud.sdk.vpc.v2.model.Subnet;
+import java.time.OffsetDateTime;
+import java.util.List;
+import lombok.Data;
+
+/**
+ * Subnet information.
+ */
+@Data
+public class SubnetInfo {
```

Comment:
no need to declare any new datatypes. We just use what is available in the Huawei SDK.